pkg/server/apiextensions.go

```diff
@@ -217,18 +224,25 @@ func (c *apiBindingAwareCRDLister) ListWithContext(ctx context.Context, selector
 			}
 
 			// Priority 2: Add APIBinding CRDs. These take priority over those from the local workspace.
+
+			// Add the APIExport identity hash as an annotation to the CRD so the RESTOptionsGetter can assign
+			// the correct etcd resource prefix.
+			crd = crd.DeepCopy()
+			crd.Annotations[apisv1alpha1.AnnotationAPIIdentityKey] = boundResource.Schema.IdentityHash
+
 			ret = append(ret, crd)
 			seen.Insert(crdName(crd))
 		}
 	}
 
-	crds, err := c.crdLister.List(labels.Everything())
+	// TODO use scoping lister when available
+	crds, err := c.crdLister.List(selector)
```

Comment:
index not worth it for now?

Reply:
Will do.

Reply:
Actually, will do later w/the wrappers 😄